jenkins_jobs/modules/triggers.py

```diff
@@ -1172,6 +1172,8 @@ def github_pull_request(registry, xml_parent, data):
         in the pull request will trigger a build (default false)
     :arg str skip-build-phrase: when filled, adding this phrase to
         the pull request title or body will not trigger a build (optional)
+    :arg str black-list-commit-author: When filled, pull request commits from this user(s)
+        will not trigger a build. (Note: this can be overridden by job-specific configuration.)
```

Comment:
I don't understand what the "Note" says

Reply:
I directly copied it over the help button on the Jenkins UI. Honestly, I don't understand it either. Should I delete it?

Reply:
I think so, I doubt it would be accepted into upstream.